### ISSUE TITLE
Task 2: Wayfinding

### ISSUE THREAD

**Issue by cbrxyz** [uf-mil/mil#1078]
### What needs to change?

From the VRX manual:

> **Capability:**
System should be capable of command and control of USV to achieve a series of given goal states, specified as a series of locations/heading values.
**Implementation:**
Teams receive a list of goal states which they are free to visit in any order. The simulation continues until the maximum time is exceeded. At each time step, the combined pose error is calculated between the USV and every waypoint using the same formula for pose error distance described in the station keeping task. The performance for each waypoint is quantified by the minimum pose error achieved for that waypoint over all time steps.

Basically, ensure that the boat is able to move to a series of waypoints in order with minimal loss in accuracy.

### How would this task be tested?

1. Run the VRX simulation environment and ensure that loss is minimal by observing the topics outlined in the VRX manual.
2.
3.

### Contacts

- [ ] We need help from the mechanical team.
- [ ] We need help from the electrical team.
- [ ] We need help from Dr. Schwartz or other faculty.
- [ ] We need help from a company or an organization.
- [ ] We need help from another UF staff member or organization (ex, facilities).

**Comment by MrinallU** [comment]
@cbrxyz Can I do this?
